hack/tools/tilt-prepare/main.go

```diff
@@ -570,6 +582,87 @@ func certManagerTask() taskFunction {
 	}
 }
 
+// kustomizationFile represents a kustomization.yaml.
+type kustomizationFile struct {
+	HelmCharts []kustomizationFileHelmCharts `json:"helmCharts"`
+}
+type kustomizationFileHelmCharts struct {
+	Name    string `json:"name"`
+	Version string `json:"version"`
+}
+
+// chartFile represents a Chart.yaml.
+type chartFile struct {
+	Version string `json:"version"`
+}
+
+// cleanupChartTask cleans up outdated charts below the given path if they don't
+// match the pinned version. This is necessary because kustomize just uses a local
+// Chart if it exists.
```

Comment:
nit
```suggestion
// cleanupChartTask ensures we are using the required version of a chart by cleaning up 
// outdated charts below the given path. This is necessary because kustomize just 
// uses a local Chart if it exists, no matter if it matches the required version or not.
```

Reply:
Thx!